files/image_config/caclmgrd/caclmgrd

```diff
@@ -152,10 +163,22 @@ class ControlPlaneAclManager(object):
                         log_error("ACL rule does not contain PACKET_ACTION property")
                         continue
 
+                    # If we haven't determined the IP version for this ACL table yet,
+                    # do it now. We determine heuristically based on whether the
+                    # src IP is a v4 or v6 address.
+                    if not table_ip_version:
+                        if "SRC_IP" in rule_props and rule_props["SRC_IP"]:
```

Comment:
if SRC_IP is not specified, does the ACL need to be applied to both iptables AND ip6tables?

Reply:
No. We assume that with regard to service ACLS, IPv4 and IPv6 ACLs will be stored in separate tables. If a rule needs to be applied to both IPv4 and IPv6, it needs to be specified twice, one in a v4 table and once in a v6 table.

However, I realize that this implementation also assumes that the first rule in the table will always have a SRC_IP defined. If not, then the table will be assumed to be v4, whether or not subsequent rules have v6 SRC_IPs. This may not always be true.